src/server/controllers/payment.controller.js

```diff
@@ -62,6 +64,10 @@ export const makePayment = async (req, res) => {
           res.redirect(`${config.urlRoot}/payment-code/${penaltyDetails.paymentCode}`);
         });
       case 'cheque':
+        if (!authorizedRoles.some(item => item === userRole.toLowerCase())) {
```

Comment:
Why are we doing this for cheque and not other transaction types?

Reply:
Cheque payment is restricted to call centre only, as documented in the permission structure.